api/envoy/api/v2/core/health_check.proto

```diff
@@ -233,29 +258,3 @@ message HealthCheck {
   // The default value is false.
   bool always_log_health_check_failures = 19;
 }
-
-// Endpoint health status.
```

Comment:
IIRC this will change the doc order, right? Moving the enums at the top level like this seems sub-optimal, but maybe there is nothing we can do about this.

Reply:
I think we're safe here, since the `protodoc` tool does not look at order in the `.proto` but only operates on the `FileDescriptorProto`, for which this should be a nop.